libpdbg/cfam.c

```diff
@@ -320,7 +320,8 @@ static int cfam_hmfsi_probe(struct pdbg_target *target)
 	int rc;
 
 	/* Enable the port in the upstream control register */
-	assert(!(pdbg_target_u32_property(target, "port", &port)));
+	rc = pdbg_target_u32_property(target, "port", &port);
+	assert(!rc);
```

Comment:
This is not correct.  We cannot continue if the port attribute is missing. Dropping this hunk from patch.